src/main/java/apitests/clients/PetClient.java

```diff
@@ -0,0 +1,26 @@
+package apitests.clients;
+
+import apitests.model.Pet;
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.given;
+
+public class PetClient extends BaseClient {
+
+    private static final String PET_URL = BASE_URL + "/pet/";
+
+    public Response getPetsByStatus(final String status) {
+        return given()
+                .when()
+                .queryParams("status", status)
```

Comment:
You can use Fields from Pet class for status name.

Reply:
Fixed